configs/sst_networking-management-server.yaml

```diff
@@ -19,10 +19,6 @@ data:
   - NetworkManager-ovs
   - nmstate-plugin-ovsdb
 
-  # Team
-  - NetworkManager-team
```

Comment:
Since these are still in Stream 9 you probably want to copy the current state into a sst_networking-management-server-c9s.yaml file and move the c9s label over there

Reply:
Oh. I thought CS9 was snapshoted a long time ago.
Ok, I'll update the PR. Thx.